server/utils/parse_helper.py

```diff
@@ -110,6 +125,47 @@ def reparse_cleanup(project_details, user_id):
     delete_folder(directory)
 
 
+def extract_repository_metadata(repo):
+    languages = repo.get_languages()
+
+    total_bytes = sum(languages.values())
+    latest_commit = repo.get_commits()[0]
+
+    metadata = {
+        "basic_info": {
+            "full_name": repo.full_name,
+            "description": repo.description,
+            "created_at": repo.created_at.isoformat(),
+            "updated_at": repo.updated_at.isoformat(),
+            "default_branch": repo.default_branch,
+        },
+        "metrics": {
+            "size": repo.size,
+            "stars": repo.stargazers_count,
+            "forks": repo.forks_count,
+            "watchers": repo.watchers_count,
+            "open_issues": repo.open_issues_count,
+            "open_pull_requests": len(list(repo.get_pulls(state='open'))),
```

Comment:
dont need this

Reply:
Again, whole thing or just open_pull_request?

Reply:
Just open pull requests